builtInFunctions/multiESDTNFTTransfer.go

```diff
@@ -508,6 +511,7 @@ func (e *esdtNFTMultiTransfer) addNFTToDestination(
 	esdtTokenKey []byte,
 	nonce uint64,
 	isReturnCallWithError bool,
+	isSenderESDTSCAddr bool,
```

Comment:
this should be refactored in the future to be more "clean code"

Reply:
Right, will create a task for it